src/foam/u2/borders/CollapseBorder.js

```diff
@@ -77,15 +82,18 @@ foam.CLASS({
         addClass(this.myClass()).
         enableClass('expanded', this.expanded$).
         start('div').
+          style({ 'display': 'flex', 'flex-direction': 'row', 'align-items': 'center' }).
           addClass(this.myClass('toolbar')).
+          start('div').
+            addClass(this.myClass('control')).
+            addClass( this.toggleLeft ? '' : this.myClass('float-right')).
+            tag(this.TOGGLE, { label: this.label$ }).
+          end().
           start('span').
+            on('click', () => this.toggle()).
```

Comment:
Why are we replacing the button with a listener? This will cause accessibility issues

Reply:
im using this for a part of the transaction plan cards, would be a nice to have if this could toggle on click of the title as well, since the button is a bit small